### PR TITLE
revise content, add security policy

### PR DESCRIPTION
**Pull request by dhuseby**
* revised the above-fold copy to better promote the accomplishments of libp2p
* added security policy at the bottom
* removed cruft in CSS

### CODE REVIEW

**Comment by dhuseby** [comment]
security

File: templates/security.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <p><b>lib</b>p2p is still under active development and there may be
+    problems with our protocols or errors in our implementations. We take
+    security vulnerabilities very seriously and follow responsible disclosure
+    best practices. If you discover a security issue, please bring it to our
+    attention right away!</p>
+  </div>
+  <h2>Reporting a Security Vulnerability</h2>
+  <div class="wrap" style="padding-bottom:30px;">
+      <p>If you find a vulnerability that may affect live deployments—such as a
+      remote execution exploit—please <a href="https://github.com/libp2p/go-libp2p/security/advisories/new">report it privately</a>.
```
Comment (MarcoPolo) [inline templates/security.html]: Should we rephrase to link to each implementation's advisories? Or maybe just say to reach out to the email and we can triage.

Comment (dhuseby) [reply templates/security.html]: I think the email is easier for a central security team to manage.

Comment (dhuseby) [reply templates/security.html]: OH! I see, yeah, the link is for go-libp2p. Hrm...I think maybe we just say, "report it privately by sending an email to security@libp2p.io. DO NOT file a public issue"

File: templates/security.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    best practices. If you discover a security issue, please bring it to our
+    attention right away!</p>
+  </div>
+  <h2>Reporting a Security Vulnerability</h2>
+  <div class="wrap" style="padding-bottom:30px;">
+      <p>If you find a vulnerability that may affect live deployments—such as a
+      remote execution exploit—please report it privately by sending an email to 
+      <a href="mailto:security@libp2p.io">security@libp2p.io</a>. <b>DO NOT</b>
+      file a public issue.</p>
+
```
Comment (dhuseby) [inline templates/security.html:16]: @MarcoPolo I changed it to direct people to send an email.

**Pull request merged**